model_compression_toolkit/exporter/model_exporter/tflite/int8_tflite_exporter.py

```diff
@@ -0,0 +1,165 @@
+            is_layer_exportable_fn: Callable to check whether a layer can be exported or not.
+            save_model_path: Path to save the exported model.
+        """
+        super().__init__(model,
+                         is_layer_exportable_fn,
+                         save_model_path)
+
+        self.exported_model = None
+
+    def export(self) -> None:
+        """
+        Export a fully quantized model to its int8 tflite model.
+        """
+
+        def _substitute_model(wrapped_layer):
+            assert self.is_layer_exportable_fn(
+                wrapped_layer), f'Layer {wrapped_layer.get_config()} did not pass validation'
+
+            # In order to support dense quantization using per-channel quantization (which is
+            # unsupported in TFLITE int models) we substitute each dense layer to its equivalent
+            # point-wise convolution.
+            if isinstance(wrapped_layer.layer, Dense):
+                # First we create a pointwise configuration based on the Dense layer's configuration
+                dense_cfg = wrapped_layer.layer.get_config()
+
+                # List of pw attributes that should take from the dense layer as they are.
+                pw_attr_list = [LAYER_NAME, ACTIVATION, USE_BIAS, BIAS_CONSTRAINT,
+                                BIAS_INITIALIZER, BIAS_REGULARIZER, TRAINABLE, ACTIVITY_REGULARIZER,
+                                KERNEL_INITIALIZER, KERNEL_REGULARIZER, KERNEL_CONSTRAINT]
+                pw_cfg = {attr: dense_cfg[attr] for attr in pw_attr_list}
+
+                # Use more attributes that are not taken as are
+                pw_cfg.update({KERNEL_SIZE: (1, 1),
+                               STRIDES: (1, 1),
+                               PADDING: PAD_VALID,
+                               FILTERS: dense_cfg[UNITS]})
+
+                # Create the point-wise layer
+                pw_layer = Conv2D(**pw_cfg)
+                pw_layer.build(wrapped_layer.layer.input_shape)
+
+                # Create and set the point-wise weights to assign
+                dense_kernel = wrapped_layer.layer.kernel
+                pw_weights = []
+                pw_kernel = np.reshape(wrapped_layer.get_weights()[0],
+                                       (1, 1, dense_kernel.get_shape()[0], dense_cfg[UNITS]))
+                pw_weights.append(pw_kernel)
+                if wrapped_layer.layer.use_bias:
+                    pw_bias = wrapped_layer.get_weights()[2]
+                    pw_weights.append(pw_bias)
+                pw_layer.set_weights(pw_weights)
+
+                # Now that we have the point-wise to replace the dense layer,
+                # we need to wrap it using qi.KerasQuantizationWrapper with a new
+                # relevant quantization dispatcher based on current dispatcher information
+
+                # Create new kernel quantizer
+                pw_kernel_quantizer_cfg = wrapped_layer._dispatcher.weight_quantizers['kernel'].get_config()
```

Comment:
Isn't there another way to access the quantizer config without accessing the dispatcher (which is marked as a protected variable) like that?